langgraph_supervisor/handoff.py

```diff
@@ -18,6 +18,11 @@ def _normalize_agent_name(agent_name: str) -> str:
     return WHITESPACE_RE.sub("_", agent_name.strip()).lower()
 
 
+def _has_multiple_content_blocks(content: str | list[str | dict]) -> TypeGuard[list[dict]]:
```

Comment:
don't think it should have list of str

Reply:
Same as above – it's due to the type of `BaseMessage.content`.